### ISSUE TITLE
Can't add env and service values to a Docker Lambda

### ISSUE THREAD

**Issue by icaspi** [DataDog/datadog-cdk-constructs#185]
## Expected Behavior
When setting up the datadog construct for a lambda the `env` and `service` are added with the extension layer and add environment variables to the lambda

## Actual Behavior
Adding extension layer is being reject because it is a packaged lambda. Deploying without the extension is working as expected but don't create the environment variables 

## Steps to Reproduce the Problem

  1. Create a CDK script
  2. Add to it function from a docker file
  3. Create a datadog component and include `extensionLayerVersion`  key
  

## Specifications

  - Datadog Lambda Layer version: 43 
  - Node version: 
  
## Stacktrace
  
  ```
 "Please don't provide Handler or Runtime or Layer when the intended function PackageType is Image. (Service: Lambda, Status Code: 400, Request ID: 0db9418c-418f-4c5b-b6a3-f89ec1d9f873)" (RequestToken: 75a65225-d578-b731-7070-0558c4690f9a, HandlerErrorCode: InvalidRequest)
    at FullCloudFormationDeployment.monitorDeployment (/usr/local/lib/node_modules/aws-cdk/lib/index.js:380:10236)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async deployStack2 (/usr/local/lib/node_modules/aws-cdk/lib/index.js:383:145458)
    at async /usr/local/lib/node_modules/aws-cdk/lib/index.js:383:128776
    at async run (/usr/local/lib/node_modules/aws-cdk/lib/index.js:383:126782)
  ```

**Comment by sfirrin** [comment]
Hi @icaspi, thanks for opening this

We will look into a fix and update you here

**Comment by sfirrin** [comment]
To update @icaspi, I have just been able to reproduce and am currently working on a fix

**Comment by sfirrin** [comment]
Hi again @icaspi, we just released https://github.com/DataDog/datadog-cdk-constructs/releases/tag/v2-1.7.4 with the fix and in 1.7.4 I no longer get the error in my testing Docker CDK app. Thanks for opening the issue and please get in touch again if you run into any other problem
